src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -2139,19 +2149,22 @@ private List<EC2AbstractSlave> provisionOndemand(
         logProvisionInfo("Looking for existing instances with describe-instance: " + diRequest);
 
         DescribeInstancesResponse diResult = ec2.describeInstances(diRequest);
-        List<Instance> orphansOrStopped = findOrphansOrStopped(diResult, number);
-
-        if (orphansOrStopped.isEmpty()
-                && !provisionOptions.contains(ProvisionOptions.FORCE_CREATE)
-                && !provisionOptions.contains(ProvisionOptions.ALLOW_CREATE)) {
-            logProvisionInfo("No existing instance found - but cannot create new instance");
-            return null;
-        }
+        List<Instance> orphansOrStopped = new ArrayList<>();
```

Comment:
More clearly, `List.of()` (or the older `Collections.emptyList()`) to emphasize that we do not intend to mutate it. Clearer still,

```java
List<Instance> orphansOrStopped;
if (!avoidUsingOrphanedNodes) {
    orphansOrStopped = findOrphansOrStopped(diResult, number);
    // …
} else {
    orphansOrStopped = List.of();
}
```